replace non-existing locale-gen with locale-def

wizardofoz123 opened the pull request:
Signed-off-by: Jakub Racek <jracek@redhat.com>

Commit:
replace non-existing locale-gen with locale-def

Signed-off-by: Jakub Racek <jracek@redhat.com>

agustinhenze commented:
Feel free to merge after the images are built
* https://quay.io/repository/cki/fedora29
* https://quay.io/repository/cki/centos7

Commit:
wokaround weird bug: reinstall pip3

Apparently there's some kind of weird bug where after doing

RUN pip2 install -U pip setuptools virtualenv

the pip3 installation is damaged and pip3 command won't be found, even
though there's a file in /usr/bin

Signed-off-by: Jakub Racek <jracek@redhat.com>
